typings/index.d.ts

```diff
@@ -8,6 +8,9 @@ declare namespace Omnibar {
   type FunctionalExtension<T> = (query: string) => Results<T>;
   type Extension<T> = FunctionalExtension<T>;
 
+  // Renderers
+  type ResultRenderer = <T>({ item }: { item: T }) => JSX.Element;
```

Comment:
ohh, I really have to get used to this #typescript

Reply:
we've been spoiled by not having to deal with types for too long. 😆